app/pages/authority/routes.test.js

```diff
@@ -0,0 +1,34 @@
+import request from 'supertest';
+import { App } from '../../../app';
+import routes from './routes';
+import * as dashboardControllers from './dashboard/controller';
+
+jest.mock('../../logger');
+
+const mockDashboardContext = {
+  title: 'Dashboard title',
+  previewUrl: '/',
+  submitForModerationUrl: '/',
+  returnToDashboardUrl: '/',
+  sectionGroups: [
+    {
+      id: 'sectiongroup-id',
+      title: 'Sectiongroup Title',
+    },
+  ],
+};
+
+describe('GET /authority/solution/:solutionId', () => {
+  it('should return the correct status and text if there is no error', () => {
```

Comment:
what about if there is an error?

Reply:
Good point. I've added a unit test to the main routes.test.js to check the error page is shown if there is an error from the /authority endpoint.